be2-scala/src/test/scala/ch/epfl/pop/model/network/method/message/data/socialMedia/AddChirpSuite.scala

```diff
@@ -7,13 +7,13 @@ import ch.epfl.pop.model.network.method.message.data.{ActionType, ObjectType}
 import spray.json._
 
 class AddChirpSuite extends FunSuite with Matchers {
-    test("Constructor/apply works as intended"){
-        val text: String = "text"
-        val parent_id: Option[String] = None
-        val timestamp = Timestamp(0)
+    private final val text: String = "text"
```

Comment:
Test suite would look better if you put `AddChirp message creation in a separate scala object (same file should be fine)

Reply:
done, as we discussed